platform/lib/schema/__tests__/__snapshots__/MapOfSetting.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fails when not receiving expected key type 1`] = `"[name]: expected value of type [number] but got [string]"`;
```

Comment:
I think it's good enough for now. I still need to learn what these jest snapshot are about :)